libr/core/cmd_mount.c

```diff
@@ -297,28 +297,60 @@ static int cmd_mount(void *data, const char *_input) {
 		break;
 	case 'g': // "mg"
 		input++;
+		int offset = 0;
+		int size = 0;
 		if (*input == ' ') {
 			input++;
 		}
 		ptr = strchr (input, ' ');
 		if (ptr) {
 			*ptr++ = 0;
+			char *input2 = strdup (ptr++);
+			const char *args = r_str_trim_head_ro (input2);
+			if (args) {
+				ptr = strchr(args, ' ');
+				if (ptr) {
+					*ptr++=0;
+					size = r_num_math(NULL, ptr);
+				}
+				offset = r_num_math(NULL, args);
+			}
 		} else {
 			ptr = "./";
 		}
-		file = r_fs_open (core->fs, input, false);
+		const char *filename = r_str_trim_head_ro(input);
+	
+		file = r_fs_open (core->fs, filename, false);
 		if (file) {
-			char *localFile = strdup (input);
+			char *localFile = strdup (filename);
 			char *slash = (char *)r_str_rchr (localFile, NULL, '/');
 			if (slash) {
 				memmove (localFile, slash + 1, strlen (slash));
 			}
-			r_fs_read (core->fs, file, 0, file->size);
-			r_file_dump (localFile, file->data, file->size, false);
+			size_t ptr = offset;
+			int total_bytes_read = 0;
+			int blocksize = file->size < core->blocksize ? file->size : core->blocksize;
+			size = size > 0 ? size : file->size;
```

Comment:
check if file exists before overwriting it. and if scr.interactive is set prompt the user with r_cons_yesno() to overwrite. Will be good to have support for -f .. or maybe /mgf

Reply:
this can be done in another pr

Reply:
I would rather do this work in another pr

Reply:
Agree